src/main/java/com/t795/trpo/Kraiko/Main.java

```diff
@@ -0,0 +1,348 @@
+package com.company;
```

Comment:
create another folder `lab4` , the full path to source will be like src/main/java/com/t795/trpo/Kraiko/lab4/Main.java